pkg/cluster/cluster_config.go

```diff
@@ -53,16 +53,25 @@ func IsNotReservedNamespace(ns *corev1.Namespace) bool {
 // GetClusterServiceVersion retries the clusterserviceversions available in the operator namespace.
 func GetClusterServiceVersion(ctx context.Context, c client.Client, watchNameSpace string) (*ofapiv1alpha1.ClusterServiceVersion, error) {
 	clusterServiceVersionList := &ofapiv1alpha1.ClusterServiceVersionList{}
-	if err := c.List(ctx, clusterServiceVersionList, client.InNamespace(watchNameSpace)); err != nil {
-		return nil, fmt.Errorf("failed listing cluster service versions: %w", err)
+	listOptions := &client.ListOptions{
+		Limit:     100,
+		Namespace: watchNameSpace,
```

Comment:
nit, but `watchNameSpace` reads a bit odd.

Reply:
Another question - why here it's `100` and in other case `500`? Any particular reason for such a difference? Couldn't we have a `const chunkSize` for both?